spec/requests/citizens/property_value_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+RSpec.describe Citizens::PropertyValuesController, type: :request do
+  describe 'citizen property value test' do
+    describe 'GET /citizens/property_value' do
+      before { get citizens_property_value_path }
+
+      it 'returns http success' do
+        expect(response).to have_http_status(:ok)
+        expect(unescaped_response_body).to include('How much is your home worth')
+      end
+    end
+  end
+
+  describe 'POST /citizens/property_value', type: :request do
+    let(:legal_aid_application) { create :legal_aid_application, :with_applicant }
+    before do
+      sign_in legal_aid_application.applicant
+      post citizens_property_value_path, params: params
+    end
+
+    context 'when a property value is entered' do
+      let(:params) { { legal_aid_application: { property_value: 123_456.78 } } }
+
+      it 'redirects to new action' do
+        expect(unescaped_response_body).to include('Navigate to')
+        # TO DO
+        # expect(response).to redirect_to(to_be_determined_path)
```

Comment:
I think using an `xit` is a better pattern (than just adding a comment), as it is a look more obvious that there is a behaviour that needs to be used in the future.